README.md

```diff
@@ -4,8 +4,16 @@ Some greasemonkey scripts that might aid in the development of openstack project
 ## Current scripts
 
 - **openstack_gerrit_zuul_status.user.js**: Provides the status of the current CI run on the OpenStack Gerrit in real time.
+- **FoxReplace.json** is text replacements configuration for https://addons.mozilla.org/en-GB/firefox/addon/foxreplace/ which
+would add some extra coloring on build logs, making them easier to read.
+We plan to convert this into a greasemonkey helper in the future to allow
+use from multiple browsers.
+
+## Installation
+
+After you install GreaseMonkey in your browser click the link below:
```

Comment:
Actually I'd recommend TamperMonkey or ViolentMonkey before GreaseMonkey because the latter since GM4 is not backwards compatible and even the main dev recommends going for the former two if you want backwards compatibility.